lib/ecto/changeset.ex

```diff
@@ -366,7 +366,7 @@ defmodule Ecto.Changeset do
       iex> changeset = cast(post, %{title: "Hello"}, [:title])
       iex> new_changeset = cast(changeset, %{title: "Foo", body: "Bar"}, [:body])
       iex> new_changeset.params
-      %{"title" => "Foo", "body" => "Bar"}
+      %{"title" => "Hello", "body" => "Bar"}
```

Comment:
the example is correct: (I'm using schemaless changeset which is app-agnostic, so you can run this code anywhere)

```elixir
iex> import Ecto.Changeset
iex> data = %{}; types = %{title: :string, body: :string}
iex> ch1 = cast({data, types}, %{title: "Hello"}, [:title])
iex> ch2 = cast(ch1, %{title: "Foo", body: "Bar"}, [:body])
iex> ch2.params
%{"body" => "Bar", "title" => "Foo"}
```